hosting.md

```diff
@@ -14,17 +17,20 @@ Some tips for hosting a great review club meeting:
   [Making a New Post](#making-a-new-post) section below for how to make a post.
 
 - Prepare a range of questions from basic to advanced. The review club is a way
-  to help new developers develop. There should be something for everyone.
+  to help new developers learn. There should be something for everyone.
 
 - In the notes and meeting, don't just talk about the mechanical code changes.
-  Other important aspects to consider are: the historical context of the PR (why
-  things in the code are the way they are currently, and what PRs have touched
-  that part of the code base), testing methodology and motivation for the PR.
+  Other important aspects to consider are:
+
+  - motivation for the PR
+  - the historical context of the PR (why things in the code are the way they
+    are currently, and what PRs have touched that part of the code base in the past)
+  - how to review the changes and testing methodology.
```

Comment:
suggestions (feel free to take none/some/all)

- advantages and disadvantages of this approach
- alternative approaches
- potential bugs reviewers might want to look out for
- fun C++ things